app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -315,7 +315,7 @@ private fun RecentlyVisitedMenu(
         onDismissRequest = { onDismissRequest() },
         modifier = Modifier
             .background(color = FirefoxTheme.colors.layer2)
-            .height(52.dp)
+            .height(64.dp) // minimum height of a menu with one item
             .scrollable(
```

Comment:
I also wondering if we need this `scrollable` modifier

Reply:
That seemed to me like a previous attempt to fix this bug. Which didn't really work.

Reply:
Right, this is also my understanding that we added these `scrollable` to attempt to fix the bug. So, I think we should try and revert some of it if it is no longer necessary if the 64.dp will make things work nicely

Reply:
Thank you!
Deleted the scrollable modifier from below but somehow missed this.
Tested with this modifier also removed and it works as expected.